common/testing/Makefile.mk

```diff
@@ -6,5 +6,6 @@ noinst_LTLIBRARIES += common/testing/libolatesting.la \
 common_testing_libolatesting_la_SOURCES = \
     common/testing/MockUDPSocket.cpp \
     common/testing/TestUtils.cpp
+common_testing_libolatesting_la_CXXFLAGS = $(COMMON_TESTING_PROTOBUF_FLAGS)
```

Comment:
We need @RenZ0 to check the exact message you were getting for this please and I'll see if we can use a more bespoke and appropriate set of flags? This will be in some time when he can restart the machine again.

Reply:
It is on FreeBSD 10.0-RELEASE. I can't reproduce that, I guess I had the message because I started from the release and I didn't run autoreconf. So this change is not needed.

Reply:
Okay, yeah great, lets revert this change then.